client/src/pages/MainTable.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from "react";
+
+const MainTable = () => {
+  console.log("Page Uploaded");
+  const [clientData, SetClientData] = useState (null);
```

Comment:
```suggestion
  const [clientData, setClientData] = useState (null);
```
The second variable is a function, and the naming convention for functions is camelCase. Can we update `SetClientData` to start with a lowercase `s`?

Reply:
Thanks Coung, already fixed.